spec/index.bs

```diff
@@ -5888,6 +5889,19 @@ dictionary GPUImageCopyTextureTagged : GPUImageCopyTexture {
         2d canvases are [[html#premultiplied-alpha-and-the-2d-rendering-context|always premultiplied]],
         while WebGL canvases can be controlled via <l spec=html>[=WebGLContextAttributes=]</l>.
         {{ImageBitmap}} premultiplication can be controlled via {{ImageBitmapOptions}}.
+
+    : <dfn>originBottomLeft</dfn>
+    ::
+        Describes whether the origin (0, 0) to copy from source image is bottom left corner, or not.
+       
+        If this option is set to `true` and the origin (0, 0) of {{GPUImageCopyExternalImage/source}} is top left corner,
+        the source values must be flipped along the vertical axis to change the origin (0, 0) to bottom left corner.
```

Comment:
Semantically, I don't think any image has a bottom-left origin. We could just as well call it `boolean flipY` if all images have a top-left origin. Even if some images had bottom-left origins in some sense, I would still consider reading from the bottom-left to be "y flipped" as they will always render from the top-left when used in, say, `drawImage()` (AFAIK).

In any case, I think the naming of `originBottomLeft` is unambiguous so the reader doesn't have to wonder about that. I would just update the text - instead of saying essentially "flip if the source corner is different", we can say something like "The source image is vertically flipped, so the copy starts from the bottom row of the source." Which is direct about where the copy starts, and slightly redundant for clarity.

(And I would avoid the use of "(0, 0)" to describe the orientation of the source image, as we already have `GPUImageCopyExternalTexture.origin` and it's ambiguous with that.)